### DISCUSSION TITLE
When routing forwarding, the destination page does not receive parameters

### DISCUSSION THREAD

**Discussion by JR199706** [nuxt/nuxt#20482] [Questions]
`
  function gotoPage() {
      // return router.push({
      //     name: 'ebook-upload',
      //     params: {
      //         id: 999,
      //     }
      // })
  
      return navigateTo({
          name: 'ebook-upload',
          params: {
              id: 999,
          }
      })
  }

  /ebook/upload
  console.log("=====",route.params.id)  // Undefined
`

**Comment by maou-shonen** [comment]
params need to use a naming convention, it looks like you didn't name your files correctly. [See docs](https://nuxt.com/docs/getting-started/routing#pages)
You can also simply modify it to use query.
